### ISSUE TITLE
tests.test_forcedsampler.ForcedSamplerTest failing

### ISSUE THREAD

**Issue by DePierre** [CANToolz/CANToolz#7]
When re-organizing the unit tests for CANToolz, I found some left-alone tests in `cantoolz/stream/forced_sampler__test.py`, which I moved to https://github.com/CANToolz/CANToolz/blob/dev/tests/test_forcedsampler.py

The test is failing and I am personally not familiar with this test yet. @eik00d Do you have memories of what the test is supposed to do? And why it is failing?

Any info would help me understand if I should remove/update/fix the code accordingly.

See: https://travis-ci.org/CANToolz/CANToolz/jobs/268744940#L488

**Comment by eik00d** [comment]
Yes, this is code by Sergey Kononeko (@kononencheg), so I am also not familiar with those tests. 
Force sampler used into experimental features like automated fields detection and extractions from CAN Frames . Will ask Sergey what to do...

**Comment by DePierre** [comment]
@eik00d Could you get any answer from Sergey? I'll dive into the code soon if he can't help us otherwise, no worries.

**Comment by eik00d** [comment]
sent request to him again...

**Comment by kononencheg** [comment]
Hi! I'll fix this code (or test) during this week!
